.github/workflows/build-release.yml

```diff
@@ -0,0 +1,74 @@
+name: Build
+on: [push, pull_request]
+
+jobs:
+  deploy:
+    name: Build dependencies & create artifact
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.0.0
```

Comment:
Il me semble qu'il faille désormais utiliser les v3 des actions génériques, sous Node 16.

Reply:
Hi @PrestaEdit, I think you're right. Thanks 🙏